tests/test_model.py

```diff
@@ -269,14 +269,16 @@ def assert_ataos_corrections(
     @staticmethod
     def get_random_value_test_config() -> typing.Dict[str, typing.List[np.ndarray]]:
         return {
-            "m1": [np.random.rand()],
-            "m2": [np.random.rand()],
-            "hexapod_x": [np.random.rand()],
-            "hexapod_y": [np.random.rand()],
-            "hexapod_z": [np.random.rand()],
-            "hexapod_u": [np.random.rand()],
-            "hexapod_v": [np.random.rand()],
-            "chromatic_dependence": [np.random.rand()],
+            # Added these ignores when converting to pyproject.toml to make
+            # mypy happy.
```

Comment:
I think the return value annotation is wrong. Try replacing `np.ndarray` with `float` because that's what `np.random.rand()` returns.

If that works, please remove the new `# type: ignore` and the new comment about mypy.

Reply:
This works. Thanks a lot! I removed the `  # type: ignore` comments as well as the other added comment.